docs/templates/config.rst.j2

```diff
@@ -40,6 +40,21 @@ Ansible will process the above list and use the first file found, all others are
         inventory = /etc/ansible/hosts  ; This points to the file that lists your hosts
 
 
+.. _generating_ansible.cfg:
+
+Generating a sample ``ansible.cfg`` file
+-----------------------------------------
+
+You can generate a fully commented-out example ``ansible.cfg`` file, for example::
```

Comment:
```suggestion
You can generate a fully commented-out example ``ansible.cfg`` file, for example:

.. code-block:: shell
```

Reply:
I tried it that way and it didn't generate the code block, so I stuck with the `::` approach.